File: 백준/Bronze/17851. This Problem’s a Slam Dunk/This Problem’s a Slam Dunk.cc
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;
int main(void) {
    int rlt=0;
    vector<int> team1(5),team2(5);
    for(int i=0;i<5;i++)
        cin>>team1[i];
    for(int i=0;i<5;i++)
        cin>>team2[i];
    sort(team1.begin(),team1.end(),less<int>());
    sort(team2.begin(),team2.end(),less<int>());
    for(int i=0;i<5;i++)
        if(team1[i]>team2[i])
            rlt+=1;
    cout<<rlt;
    return 0;
}

File: 백준/Bronze/17851. This Problem’s a Slam Dunk/This Problem’s a Slam Dunk.py
import sys
input=lambda:sys.stdin.readline().rstrip()
team1=sorted([*map(int,input().split())])
team2=sorted([*map(int,input().split())])
rlt=0
for i in range(5):
    if team1[i]>team2[i]:
        rlt+=1
print(rlt)